build.gradle

```diff
@@ -1,570 +1,41 @@
 //
 // build.gradle
 //
-// Copyright (c) 2017, 2018, 2019 Couchbase, Inc All rights reserved.
+// Copyright (c) 2018 Couchbase, Inc.  All rights reserved.
 //
-// Licensed under the Apache License, Version 2.0 (the "License");
+// Licensed under the Couchbase License Agreement (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
+// https://info.couchbase.com/rs/302-GJY-034/images/2017-10-30_License_Agreement.pdf
 //
 // Unless required by applicable law or agreed to in writing, software
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-import org.apache.tools.ant.filters.ReplaceTokens
-import org.gradle.internal.os.OperatingSystem
-
-
-// ----------------------------------------------------------------
-// Build Script
-// ----------------------------------------------------------------
-
 buildscript {
     ext {
+        COMPILE_SDK_VERSION = 29
+        BUILD_TOOLS_VERSION = "29.0.2"
         KOTLIN_VERSION = '1.3.50'
-        JAVA_HOME = org.gradle.internal.jvm.Jvm.current().javaHome
+        JACOCO_VERSION = '0.8.4'
```

Comment:
I don't think it will hurt anything to define the JACOCO version up here and I think that other stuff needs it.

Reply:
I didn't change anything here, basically carrying over from your change.

Reply:
Oh... ok.  I should have put that JACOCO version in there...